backend-api/tests/infra-tests/application.test.ts

```diff
@@ -331,6 +331,30 @@ describe("Backend application infrastructure", () => {
     });
   });
 
+  it("apiGateway WellKnown5XXHighThresholdAlarm should have a runbook if alarm is enabled", () => {
```

Comment:
Please may we also have the low threshold alarm for the eventuality that someone marks it as critical by mistake?

Reply:
Please may we add a new describe block for Alarms at the top level of this test file?